timescaledb/how-to-guides/multinode-timescaledb/multinode-administration.md

```diff
@@ -0,0 +1,257 @@
+authentication][multi-node-authentication] for more information.
+
+Some roles might also be configured without the `LOGIN` attribute on
+the access node so that one can switch to the role locally but not
+connect with the user from a remote location. However, to be able to
+connect from the access node to a data node as that user, the data
+nodes need to have the role configured with the `LOGIN` attribute
+enabled. To create a non-login role for a multi-node setup, issue the
+following commands:
+```sql
+CREATE ROLE alice WITHOUT LOGIN;
+CALL distributed_exec($$ CREATE ROLE alice WITH LOGIN; $$);
+```
+
+To allow a new role to create distributed hypertables it also needs to
+be granted usage on data nodes:
+```sql
+GRANT USAGE ON FOREIGN SERVER dn1,dn2,dn3 TO alice;
+```
+
+By granting usage on some data nodes, but not others, it is possible
+to restrict usage to a subset of data nodes based on the role.
+
+### Altering a distributed role
+When altering a distributed role, the same process is followed as when
+creating roles. The role needs to altered locally and on the data
+nodes in two separate steps. For example, to add the `CREATEROLE`
+attribute to a role do:
+
+```sql
+ALTER ROLE alice CREATEROLE;
+CALL distributed_exec($$ ALTER ROLE alice CREATEROLE; $$);
+```
+
+## Managing distributed databases
+A distributed database can contain both distributed and
+non-distributed objects. In general, when a command is issued to alter
+a distributed object it applies to all nodes that have that object (or
+a part of it).
+
+However, in some cases settings *should* be different depending on
+node, because nodes might be provisioned differently (having, e.g.,
+varying levels of CPU, memory, and disk capabilities) and the role of
+the access node is different from a data node's.
+
+This section describes how and when commands on distributed objects
+are applied across all data nodes when executed from within a
+distributed database.
+
+### Altering a distributed database
+An [`ALTER DATABASE`][alter-database] command is only applied locally on the access
+node. There are several reasons why TimescaleDB does *not* apply this
+command across all data nodes.
+
+First, a distributed database supports having different database names
+on each data node. Therefore, renaming a distributed database on the
+access node is a local operation that applies only to the access node
+database. Note that when renaming a data node database the
+configuration of the data node must also be updated on the access
+node.
+
+
+Second, database-level configuration parameters should often be
+different across nodes. For example:
+
+```sql
+ALTER DATABASE mydatabase SET max_parallel_workers TO 12;
+```
+
+is a setting that might differ depending on the CPU capabilities of
+the node.
+
+Third, a database option like:
+
+```sql
+ALTER DATABASE mydatabase WITH CONNECTION_LIMIT 10;
+```
+
+is also, in most cases, different on an access node compared to its
+data nodes.
```

Comment:
```suggestion
Additionally, database-level configuration parameters often need to be
different across nodes. For example, this is a setting that might differ 
depending on the CPU capabilities of the node:
```sql
ALTER DATABASE mydatabase SET max_parallel_workers TO 12;
```

It is also possible that a database option like is different on an access node compared to its
data nodes:
```sql
ALTER DATABASE mydatabase WITH CONNECTION_LIMIT 10;
```


```